From b982a04cc01d5f6c985c1e5dd701383fb328bc93 Mon Sep 17 00:00:00 2001
From: Keunhong Park <keunhong@codeaurora.org>
Date: Mon, 4 Jun 2012 13:43:45 -0600
Subject: [PATCH] Change PatchTable pointer when loading patch

Originally when navigating between patch screens the highlighted row
(pointer) of the file list table would not change.

This patch updates the file list table when a new patch screen is loaded
so that the pointer corresponds to the current patch being displayed.

Change-Id: I8944062ab7abda0951b30b304aaadfd5781c92a8
---
 .../google/gerrit/client/patches/PatchScreen.java  |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/gerrit-gwtui/src/main/java/com/google/gerrit/client/patches/PatchScreen.java b/gerrit-gwtui/src/main/java/com/google/gerrit/client/patches/PatchScreen.java
index ffc8960..b1a7fec 100644
--- a/gerrit-gwtui/src/main/java/com/google/gerrit/client/patches/PatchScreen.java
+++ b/gerrit-gwtui/src/main/java/com/google/gerrit/client/patches/PatchScreen.java
@@ -297,6 +297,9 @@ public abstract class PatchScreen extends Screen implements
   @Override
   protected void onLoad() {
     super.onLoad();
+
+    fileList.movePointerTo(patchKey);
+
     if (patchSetDetail == null) {
       Util.DETAIL_SVC.patchSetDetail(idSideB,
           new GerritCallback<PatchSetDetail>() {
-- 
1.7.8.3

